tasks/rh-sign-image-cosign/rh-sign-image-cosign.yaml

```diff
@@ -85,7 +85,7 @@ spec:
             attempt=0
             until [ "$attempt" -gt "$(params.retries)" ] ; do # 3 retries by default
                 cosign -t 3m0s sign\
-                ${COSIGN_COMMON_ARGS}\
+                "${COSIGN_COMMON_ARGS}"\
```

Comment:
It reported "syntax issue" after the change. 

_[e2e-test]   + cosign -t 3m0s sign '--tlog-upload=false --key awskms:///xxx' --sign-container-identity registry....io/rhtap/konflux-release-e2e:latest quay.io/redhat../rhtap----konflux-release-e2e@sha256:bf2fb2c7d63c924ff9170c27f0f15558f6a59bdfb5ad9613eb61d3e4bc1cff0a
[e2e-test]   Error: invalid argument "false --key awskms:///xxx" for "--tlog-upload" flag: strconv.ParseBool: parsing "false --key awskms:///xxx": invalid syntax_

Reply:
Change to
 ```
COSIGN_COMMON_ARGS=(
  --tlog-upload=false
  --key “$SIGN_KEY”
)
```  
then
`cosign -t 3m0s sign "${COSIGN_COMMON_ARGS[@]}" `

Reply:
Good catch!

Reply:
very nice catch! thank you